README.md

```diff
@@ -1 +1,111 @@
-# ToDoList
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+ <a href="https://imgbox.com/ftUpMLFI" target="_blank"><img src="https://images2.imgbox.com/fd/29/ftUpMLFI_o.png" alt="personal logo"/ width="220"  height="auto"></a>
+  <br/>
+ </div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+
+# 📖 To Do List <a name="about-project"></a>
+
+This is the second project of the Microverve program corresponding to module two.
+"To Do List" is a simple website that displays a simple list of tasks and allows you to add and remove tasks from that list. By building this application, I learned how to manage data using JavaScript and ES6 and webpack.
+
+
+## 🛠 Built With <a name="built-with"></a>
++ Git
++ HTML
++ CSS
++ Javascript
++ ES6
++ +WebPack
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [[Github-pages]](https://sanieni6.github.io/ToDoList/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+In order to run this project you need:
++ A computer with an internet connection
++ A web browser
+
+### Setup
+Clone this repository to your computer
+
```

Comment:
- [ ] It would be nice if you add the following commands in this section to make your readme file professional, descriptive, and more professional :blush:

```sh
git clone git@github.com:sanieni6/ToDoList.git
cd ToDoList
```